README.md

```diff
@@ -102,6 +102,7 @@ make deploy
 ### Current issues with the development
 We're currently in the process of transitioning into this web client from Scratch's existing structure. As we transition, there are going to be some issues along the way that relate to how this client needs to interact with the existing infrastructure to work properly in production.
 
+#### FALLBACK ####
```

Comment:
Yep, I had thought to go with `OLD` and `NEW`, but that didn't seem right, either. `NORMAL` and `YOU` seemed to be putting too much blame or guilt.. writing code comments that you're actually presenting isn't easy :)